rest/model/commit_queue.go

```diff
@@ -88,13 +91,45 @@ func ParseGitHubComment(comment string) GithubCommentCqData {
 		if index == 1 {
 			data = parseFirstLine(line)
 		} else if index == 2 {
-			data.MessageOverride = line
+			data.MessageOverride = wrapString(line, commitMessageLineLength)
 		}
 	}
 
 	return data
 }
 
+// wrapString manually splits a string into smaller chunks and appends them to a new string.
+func wrapString(str string, limit int) string {
+	var b strings.Builder
+	currentLine := ""
```

Comment:
nit: we can move `currentLine` into the outer for loop since it's not used at the function level. Would also avoid the need to reset `currentLine` on L127.

2nd nit: the default value of a string type is already the empty string, so we could declare this as `var currentLine string`.